schema_utils.py

```diff
@@ -155,14 +155,23 @@ def normalize_against_schema(obj, schema, apply_custom_validators=True):
             for validator in schema[SCHEMA_KEY_VALIDATORS]:
                 kwargs = dict(validator)
                 del kwargs['id']
-                assert _Validators.get(
+                assert get_validator(
                     validator['id'])(normalized_obj, **kwargs), (
                         'Validation failed: %s (%s) for object %s' % (
                             validator['id'], kwargs, normalized_obj))
 
     return normalized_obj
 
 
+def get_validator(validator_id):
+    """Returns a validator function.
+
+    This is a necessary abstraction for backend testing. Simply returns
```

Comment:
Remove blank line above it and write briefly about the method. Returns part can be covered under `Returns` section so, no need to rewrite it.

Reply:
Addressed in most recent commit!